### PR TITLE
PLAT-1068 | Add info about not getting new album when 50% discount applied.

### PR DESCRIPTION
**Pull request by bognix**
Run `php artisan migrate && php artisan data-migration:coupon-fill-kinds`

### CODE REVIEW

File: resources/views/payment/confirm-order.blade.php [hunk 1]
```diff
@@ -80,11 +88,13 @@
 			</section>
 		@endif
 
-		<section class="subsection">
-			<p class="has-text-centered edit-personal-data">
-				<a href="{{ route('payment-personal-data') }}?edit=true">@lang('payment.confirm-change-order')</a>
-			</p>
-		</section>
+		@if($user->invoice || empty($order->coupon) || $order->coupon->kind !== \App\Models\Coupon::KIND_PARTICIPANT)
```
Comment (bognix) [inline resources/views/payment/confirm-order.blade.php]: combine with previous if statement.

File: database/migrations/2019_03_11_145341_add_kind_to_coupons.php [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+{
+	/**
+	 * Run the migrations.
+	 *
+	 * @return void
+	 */
+	public function up()
+	{
+		Schema::table('coupons', function (Blueprint $table) {
+			$table->enum('kind', ['group', 'voucher', 'participant', 'study_buddy'])->nullable();
```
Comment (gitkuba) [inline database/migrations/2019_03_11_145341_add_kind_to_coupons.php]: Do we want to create a new migration for each new kind of a coupon?

File: resources/views/payment/confirm-order.blade.php [hunk 3]
```diff
@@ -40,52 +40,59 @@
...
+						<p class="margin top big">@lang('payment.confirm-personal-data-address')</p>
+						<ul>
+							<li>{{ $user->userAddress->recipient }}</li>
+							<li>{{ $user->userAddress->street }}</li>
+							<li>{{ $user->userAddress->zip }}, {{ $user->userAddress->city }}</li>
+						</ul>
+					</div>
+				</section>
+			@endif
+			<section>
```
Comment (gitkuba) [inline resources/views/payment/confirm-order.blade.php]: else

File: resources/views/payment/confirm-order.blade.php [hunk 4]
```diff
@@ -40,52 +40,59 @@
...
-					<li>{{ $user->userAddress->recipient }}</li>
-					<li>{{ $user->userAddress->street }}</li>
-					<li>{{ $user->userAddress->zip }}, {{ $user->userAddress->city }}</li>
-				</ul>
-			</div>
-		</section>
-
-		@if($user->invoice)
+		@if($user->invoice || empty($order->coupon) || $order->coupon->kind !== \App\Models\Coupon::KIND_PARTICIPANT)
+			@if (empty($order->coupon) || $order->coupon->kind !== \App\Models\Coupon::KIND_PARTICIPANT)
```
Comment (gitkuba) [inline resources/views/payment/confirm-order.blade.php]: Simplify?

File: database/migrations/2019_03_11_145341_add_kind_to_coupons.php (hunk 2 above)
Comment (bognix) [reply database/migrations/2019_03_11_145341_add_kind_to_coupons.php]: for now it's fine. Enum is safer than simple string. Once we feel the need to create a different coupon type we can decide what to do with this.

**Pull request closed without merging**